### PR TITLE
Fix: update project info for NuGet pack

### PR DESCRIPTION
**Pull request by richardwerkman**
To prepare the codebase for automatic releases trough appveyor. The nuget pack and nuget push command will use the info from the .csproj files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
 *William Stryker: Oh, they serve their purpose... as long as they can be controlled.*
 
 ## Introduction
 
-For an introduction to mutation testing and Stryker's features, see [stryker-mutator.io](https://stryker-mutator.io/).
+For an introduction to mutation testing and Stryker's features, see [stryker-mutator.io](https://stryker-mutator.io/). Looking for [mutation testing in JavaScript](https://stryker-mutator.github.io)?
 
 ## Getting started
-Stryker.NET offers you mutation testing for your .NET Core projects. It allows you to test your tests by temporarily inserting bugs.
+Stryker.NET offers you mutation testing for your .NET Core projects. It allows you to test your tests by temporarily inserting bugs. Stryker.NET is installed using [NuGet](https://www.nuget.org/packages/Stryker.Runner.DotNetCore.CLI/).
```
Comment (nicojs) [inline README.md]: This link is not in sync with the names used in `Install`

File: src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj [hunk 2]
```diff
@@ -4,12 +4,22 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-    <PackageId>dotnet-stryker</PackageId>
-    <Authors>Richard</Authors>
-    <Company>InfoSupport</Company>
```
Comment (nicojs) [inline src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj]: You removed this. Maybe we should keep it. Better for visibility.

File: src/Stryker.Core/Stryker.Core/Stryker.Core.csproj [hunk 3]
```diff
@@ -2,19 +2,26 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Authors>Richard</Authors>
-    <Company>InfoSupport</Company>
```
Comment (nicojs) [inline src/Stryker.Core/Stryker.Core/Stryker.Core.csproj]: See previous remark

File: src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj (hunk 2 above)
Comment (richardwerkman) [reply src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj]: The package id and author were moved to below. The company attribute was removed becouse this isn't directly an Info Support package. Or would you say the Company attribute should stay?

**Comment by richardwerkman** [comment]
I've turned off .scproj file patching in the build. But somehow it keeps patching this pull request. Thats why the pr build keeps failing.

File: src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj (hunk 2 above)
Comment (nicojs) [reply src/Stryker.CLI/Stryker.CLI/Stryker.CLI.csproj]: Yeah.. maybe. What do you think @simondel ?

**Pull request merged**